tutorials/graph-theory/binary-search-tree.md

```diff
@@ -1,12 +1,391 @@
 ---
-title: 'Binary Search Tree'
-description: 'Coming Soon!'
+title: "Binary Search Tree (BST)"
+description: "Author: @utkarsh"
```

Comment:
this's not description for this topic. it should be related to BST instead. check out other tutorials for reference.

Reply:
@wingkwong  i added review it once if you want something else please tell me i will update it.